packages/react-query/src/useQuery.ts

```diff
@@ -59,6 +59,34 @@ export function useQuery<
   > & { initialData: TQueryFnData | (() => TQueryFnData) },
 ): DefinedUseQueryResult<TData, TError>
 
+export function useQuery<
```

Comment:
overload ordering matters, so please put these two as the very last overloads.

Reply:
Missed this! Just moved them to be the last overloads.